examples/tensorflow/question-answering/run_qa.py

```diff
@@ -706,7 +706,7 @@ def compute_metrics(p: EvalPrediction):
             model.compile(optimizer=optimizer, jit_compile=training_args.xla, metrics=["accuracy"])
 
         else:
-            model.compile(optimizer=None, jit_compile=training_args.xla, metrics=["accuracy"])
+            model.compile(optimizer="sgd", jit_compile=training_args.xla, metrics=["accuracy"])
```

Comment:
Can you add a comment here saying optimizer value doesn't mattr as it won't be used?

Reply:
Done!